src/ducks/transactions/TransactionModal.jsx

```diff
@@ -146,6 +183,38 @@ const TransactionModalInfoContent = withTransaction(props => {
     stackPush(<TransactionCategoryEditorSlide transaction={transaction} />)
   }
 
+  const [applicationDateBusy, setApplicationDateBusy] = useState(false)
+
+  const handleAfterUpdateApplicationDate = (_, applicationDate) => {
```

Comment:
What's this unused `_`?

Reply:
Good catch, this was a relicate of previous stuff. I changed this in https://github.com/cozy/cozy-banks/pull/1680/commits/76bae719d24a28f51ace0e01794eb0ada60cb87f, hopefully it is clearer.